modules/ROOT/pages/release-notes.adoc

```diff
@@ -25,21 +25,21 @@ video::mZ1D5Bgla4U[youtube, width=640,height=380]
 
 ==== Bonita Studio Project Composition
 
-Automation projects now use the power Maven projects. You are now able to easily compose your project of extensions (connectors, Rest API extensions, themes, and actor filters). +
+Automation projects now use the power of Maven. You can now compose your project of extensions (connectors, Rest API extensions, themes, and actor filters). +
 
-All these extensions, which are technical components, have to be built outside the Studio with our SDK. This will allow the developer to use their preferred tooling. It will also help the industrialization of your project as you will be able to implement tests for each extension making your project more robust. +
+All these extensions, which are technical components, have to be built outside the Studio with our SDK (_expect for Rest API extensions that can still be developed in the Studio_). This will allow the developer to use their preferred tooling. It will also help the industrialization of your project as you will be able to implement tests for each extension making your project more robust. +
```

Comment:
```suggestion
All these extensions, which are technical components, have to be built outside of the Studio with our SDK (_expect for Rest API extensions that can still be developed in the Studio_). This will allow the developer to use their preferred tooling. It will also help the industrialization of your project as you will be able to implement tests for each extension making your project more robust. +
```